metabase/metabase
# Add Inter font

Epic https://github.com/metabase/metabase/issues/22582

Adds a new built-in font - `Inter`.

How to test:
- Run Metabase EE
- Go to Admin -> Settings -> Whitelabel
- Make sure it is possible to select `Inter` for the font
- Make sure the UI is updated afterwards

<img width="785" alt="Screenshot 2022-07-14 at 13 23 01" src="https://user-images.githubusercontent.com/8542534/178961475-ca688412-3af9-44f8-8b18-65acf4403d6b.png">
